src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothLeAdapterImpl.cs

```diff
@@ -401,32 +401,12 @@ internal IList<BluetoothLeServiceData> GetScanResultServiceDataList(BluetoothLeS
                 _list.Add(BluetoothUtils.ConvertStructToLeServiceData(svc));
             }
 
-            Interop.Libc.Free(serviceListArray);
+            Interop.Bluetooth.FreeServiceDataList(serviceListArray, _serviceListCount);
             Marshal.FreeHGlobal(scanDataStruct.AdvData);
```

Comment:
It seems double free 
It freed on 390 line

Reply:
390 line is fail case. so, throw exception after free.